File: publish.go
package main

import (
	"context"
	"github.com/go-redis/redis/v8"
)

func main() {
	ctx := context.Background()

	rdb := redis.NewClient(&redis.Options{
		Addr:	  "localhost:6379",
		Password: "sOmE_sEcUrE_pAsS", // no password set
		DB:		  0,  // use default DB
	})

	err := rdb.Publish(ctx, "channels.order.purchase", "payload").Err()
	if err != nil {
		panic(err)
	}


}

File: publish.js
"use strict";
const ChannelsMiddleware = require("@moleculer/channels").Middleware;
const { ServiceBroker } = require("moleculer");
const faker = require("faker");

let c = 1;

// Create broker
const serviceBroker = new ServiceBroker({
	logLevel: {
		CHANNELS: "debug",
		"**": "info"
	},
    middlewares: [
		ChannelsMiddleware({
            adapter: `redis://default:sOmE_sEcUrE_pAsS@localhost:6379/0`,
			prefix: "orders", // for set prefix
		})
	]

});


serviceBroker
	.start()
	.then(async () => {
		
        console.log("Publish 'payment.processed' message...");
        
		await serviceBroker.sendToChannel("payment.processed", {
			id: faker.random.uuid(),
			name: faker.name.findName(),
			auth: faker.random.boolean(),
		})
		serviceBroker.stop();
    })
    .catch(err => {
		serviceBroker.logger.error(err);
		serviceBroker.stop();
	});